x-pack/plugins/security_solution/public/common/components/event_details/response_actions_view.tsx

```diff
@@ -36,61 +65,69 @@ export const useResponseActionsView = <T extends object = JSX.Element>({
   ecsData?: Ecs | null;
   rawEventData: SearchHit | undefined;
 }): EuiTabbedContentTab | undefined => {
+  const viewData = useMemo(
+    () => ({
+      id: EventsViewType.responseActionsView,
+      'data-test-subj': 'responseActionsViewTab',
+      name: i18n.RESPONSE_ACTIONS_VIEW,
+    }),
+    []
+  );
   const responseActionsEnabled = useIsExperimentalFeatureEnabled('endpointResponseActionsEnabled');
   const expandedEventFieldsObject = rawEventData
     ? (expandDottedObject((rawEventData as RawEventData).fields) as ExpandedEventFieldsObject)
     : undefined;
 
   const responseActions =
     expandedEventFieldsObject?.kibana?.alert?.rule?.parameters?.[0].response_actions;
-  const shouldEarlyReturn = !rawEventData || !responseActionsEnabled || !responseActions?.length;
+  const shouldEarlyReturn = !rawEventData || !responseActionsEnabled;
 
   const alertId = rawEventData?._id ?? '';
+  const [isLive, setIsLive] = useState(false);
 
   const { data: automatedList, isFetched } = useGetAutomatedActionList(
     {
       alertIds: [alertId],
     },
-    { enabled: !shouldEarlyReturn }
+    { enabled: !shouldEarlyReturn, isLive }
```

Comment:
Is the usage of the comma operator accidental here? I stumbled over this and am not sure why a comma operator is used instead of a more descriptive logical operation.

Reply:
Oh I think you just got confused here :P It's not a logical operation, it's an object with 2 fields: `enabled` and `isLive` . 
Hope that this makes sense? :)